tests/conformance_ruby/tags/include_tag_test.rs

```diff
@@ -81,33 +82,41 @@ fn test_include_tag_for() {
 }
 
 #[test]
-#[should_panic] // liquid-rust#237
+fn test_include_tag_with_single_local_variable() {
```

Comment:
Unless this was a test copied over from Ruby, this should instead just live with the tag.

Reply:
I renamed `test_include_tag_with_local_variables` to `test_include_tag_with_single_local_variables` to differeniate between testing for a single varaible and multiple. (master source: https://github.com/cobalt-org/liquid-rust/blob/dc171578116c5ba0f0a9546a02b445c7f0e0b568/tests/conformance_ruby/tags/include_tag_test.rs#L84)

I can move the test into the tag if you want though.

Reply:
Rename or fork?  I'm still seeing a test by that name.

I'd recommend moving it.  The `conformance_ruby` folder is a direct port of tests from the Ruby version of Liquid and is meant to be untouched except for
- Removing panics
- Updating to better reflect upstream